docker-compose.yml

```diff
@@ -5,69 +5,74 @@ services:
       network: host
       context: ./
       args:
-        CONFIG: src/Lachain.Console/config0.json
-        WALLET: src/Lachain.Console/wallet0.json
+        CONFIG: src/Lachain.Console/poc/config01.json
```

Comment:
There is no such file in git. Do we need this docker files changes?

Reply:
added the POC folder in the repo... we need changes such that every peer in the network is having unique RPC port